### PR TITLE
[New Rule] Modification of the Dynamic Linker Preload Shared Object

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/920

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/linux/privilege_escalation_ld_preload_shared_object_modif.toml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+author = ["Elastic"]
+description = """
+Identifies the modification of the dynamic linker Preload shared object (ld.so.preload). Adversaries may execute their
+own malicious payloads by hijacking the dynamic linker used to load libraries.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Modification of the Dynamic Linker Preload Shared Object"
```
Comment (bm11100) [inline rules/linux/privilege_escalation_ld_preload_shared_object_modif.toml:16]:
```suggestion
name = "Modification of Dynamic Linker Preload Shared Object"
```

File: rules/linux/privilege_escalation_ld_preload_shared_object_modif.toml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2021/01/27"
+maturity = "production"
+updated_date = "2021/01/27"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the modification of the dynamic linker Preload shared object (ld.so.preload). Adversaries may execute their
+own malicious payloads by hijacking the dynamic linker used to load libraries.
```
Comment (bm11100) [inline rules/linux/privilege_escalation_ld_preload_shared_object_modif.toml:10]:
```suggestion
Identifies modification of the dynamic linker preload shared object (ld.so.preload). Adversaries may execute malicious payloads by hijacking the dynamic linker used to load libraries.
```

Comment (bm11100) [reply rules/linux/privilege_escalation_ld_preload_shared_object_modif.toml:10]: re-lint

**Review by bm11100 — APPROVED** [review]

**Review by devonakerr — APPROVED** [review]

File: rules/linux/privilege_escalation_ld_preload_shared_object_modif.toml [hunk 3]
```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2021/01/27"
+maturity = "production"
+updated_date = "2021/01/27"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies modification of the dynamic linker preload shared object (ld.so.preload). Adversaries may execute malicious payloads by hijacking the dynamic linker used to load libraries.
```
Comment (bm11100) [inline rules/linux/privilege_escalation_ld_preload_shared_object_modif.toml:9]: re lint

Comment (Samirbous) [reply rules/linux/privilege_escalation_ld_preload_shared_object_modif.toml]: done, all rules will be relinted prior to merging anyway.

**Pull request merged**